sys/linux/futex.txt.const

```diff
@@ -0,0 +1,14 @@
+# Code generated by syz-sysgen. DO NOT EDIT.
```

Comment:
Please also delete all old sys/linux/futex_*.const files

and remove this line:
https://github.com/google/syzkaller/blob/master/sys/syz-extract/extract.go#L269
it now can be generated on upstream/linux-next, right?

Reply:
@dvyukov I removed old .const files.

yes, all definitions can be found in mainline kernel, so it can be enabled again, I remove those lines from extract.go